src/main.js

```diff
@@ -22,11 +22,23 @@ const checkValidationOfValues = envSpecString => {
   //element[0] is the variable e.g. ADMIN_EMAIL
   //element[1] is the type e.g. test
   envValArray = envValArray.filter(element => {
-    //check for valid variables AND types
+    //check for valid variables AND types or restricted choices
     if (
       element[0].match(alphanumericThatDoesNotStartWithDigit) &&
       validTypes.includes(element[1])
     ) {
+      //in case value has valid type
+      element[2] = 0; //0 constant that declares what should be printed for an element with type
```

Comment:
Changing the value of the argument provided in a `filter` function (in our case this is `element`), is not a good practice. We should:

1. Find another way to store this information
2. We should prefer something more human-friendly than numbers (0, 1) to represent that kind of information